generators/config/templates/default.yml

```diff
@@ -56,6 +56,29 @@
  tb_idle_timeout: '180'
  tb_record_screenshot: false
 
+ # -BrowserStack
+ bs_user_name: bs_user
+ bs_password: bs_password
```

Comment:
api key not needed here. As u can see the start_url combines from username and password.